app/classifier/index.cjsx

```diff
@@ -49,10 +75,15 @@ Classifier = React.createClass
       @loadSubject subject
     if nextProps.classification isnt @props.classification
       @prepareToClassify nextProps.classification
+    @context.geordi?.remember subjectID: @props.subject?.id
```

Comment:
The intent is that it ensures that subject-related clicks have the subject ID stored with them (prior to adding this line, they didn't). 

The data seems OK in Geordi.

I think we are covered here for the `forget`: https://github.com/zooniverse/Panoptes-Front-End/blob/909f9b824f276febb57b0d6a24c012185432ef77/app/classifier/index.cjsx#L57